src/SurveyService.Models.Dto/Requests/Question/CreateGroupQuestionRequest.cs

```diff
@@ -0,0 +1,25 @@
+﻿using LT.DigitalOffice.SurveyService.Models.Dto.Requests.Option;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace LT.DigitalOffice.SurveyService.Models.Dto.Requests.Question;
+
+public record CreateGroupQuestionRequest
+{
+  [Required]
+  public string Content { get; set; }
+  [Required]
+  public bool IsAnonymous { get; set; }
+  [Required]
+  public bool IsRevoteAvaible { get; set; }
+  [Required]
+  public bool IsObligatory { get; set; }
+  [Required]
+  public bool IsPrivate { get; set; }
+  [Required]
+  public bool HasMultipleChoice { get; set; }
+  [Required]
+  public bool HasCustomOptions { get; set; }
+  public List<CreateOptionRequest> Options { get; set; }
```

Comment:
you can't use existing CreateOptionRequest here: QuestionId in it can't be null

Reply:
Also, without sending a QuestionID - we don't get null - we get Guid.Empty, and then, in the mapper where we generate this questionID, we put it in the options, because in db we must write it for each option

Reply:
Done